owncloud/phoenix
# Redirect to private link after log in

## Description
If login page is triggered after trying to access private link, url of private link gets saved to store and via vuex-persist is persisted during the redirection to different page due to login. After login is successful the redirect to private link is triggered. During that redirection the private link gets deleted from store to allow normal log in later.

## Motivation and Context
When users don't get redirected to private link after login it is not only a bug but also could lead to potential confusion when they'd think they landed into the folder and could start throwing items in there which where supposed to be somewhere else.

## How Has This Been Tested?



- test environment: Manually
1. Create a private link for folder
2. Exit Phoenix
3. Enter private link
4. Go through authorisation process
5. Get redirected to the item
6. Copy different private link
7. Enter the private link without exiting Phoenix
8. Get redirected to the item
9. Exit Phoenix
10. Go through authorisation process without private link
11. See normal files list

## Types of changes


## Checklist:
